index.html

```diff
@@ -507,7 +512,66 @@ <h3>StatusList2021Credential</h3>
 <a href="#bitstring-encoding"></a>.
               </td>
             </tr>
-
+            <tr>
+              <td>
+                credentialStatus.ttl
+              </td>
+              <td>
+                The <code>ttl</code> indicates the "time to live" in milliseconds.
+                This property MAY be present, but if not present, implementers MUST 
+                use a value of <code>300000</code> for this property.  A verifier
+                MUST NOT use a cached <code>StatusList2021Credential</code> that was
+                cached more for more than the <code>ttl</code> duration prior to the
+                start of verification operation on a <a>verifiable credential</a>.
+                Implementations that publish the status list SHOULD align
+                any protocol-specific caching information, such as the 
+                HTTP `Cache-Control` header, with the value in this field.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                credentialStatus.size
+              </td>
+              <td>
+                The <code>size</code> indicates the size of the status entry in bits.
+                If size is omitted then the size of each status entry is assumed to be
+                1 bit (and therefore capable of capturing a boolean status).
+                <code>size</code> MAY be provided, but if <code>size</code> is not present
+                as a property of the <code>credentialStatus</code> then <code>size</code>
+                MUST be processed as 1.  <code>size</code> MUST be greater than zero.
+                if <code>size</code> is provided, and is greater than 1, then the property
+                <code>credentialStatus.message</code> MUST be present, and the number of
+                status messages must equal the number of possible values.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                credentialStatus.message
```

Comment:
Perhaps this should be "credentialStatus.enumeration"?

```suggestion
                credentialStatus.enumeration
```

Reply:
Not opposed to this "naming is hard"